docs/configuring-swc.md

```diff
@@ -192,47 +199,60 @@ Note: This feature requires `v1.2.13+`.
 
 ### jsc.transform.react
 
-- `runtime`
+#### jsc.transform.react.runtime
+
+Possible values: `automatic`, `classic`. This affects how JSX source code will be compiled.
 
-Possible values: `automatic`, `classic`.
+- Use `runtime: automatic` to use a JSX runtime module (e.g. `react/jsx-runtime` introduced in React 17).
+- Use `runtime: classic` to use `React.createElement` instead - with this option, you must ensure that `React` is in scope when using JSX.
 
-Use `automatic` to use new jsx runtimes.
+See https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html
 
-- `importSource`
+#### jsc.transform.react.importSource
 
-Only for `runtime: automatic`. `react` by default.
+When using `runtime: automatic`, determines the runtime library to import.
+
+Defaults to `react`.
 
 This option can be overrided with `@jsxImportSource foo`.
 
-Determines the runtime library to import.
+#### jsc.transform.react.pragma
+
+Replace the function used when compiling JSX expressions.
 
-- `pragma`
-  Replace the function used when compiling JSX expressions.
+Defaults to `React.createElement`.
 
 This option can be overrided with `@jsx foo`.
 
-Defaults to `React.createElement`.
+#### jsc.transform.react.pragmaFrag
 
-- `pragmaFrag`
-  Replace the component used when compiling JSX fragments.
+Replace the component used when compiling JSX fragments.
+
+Defaults to `React.Fragment`
 
 This option can be overrided with `@jsxFrag foo`.
 
-Defaults to `React.Fragment`
+#### jsc.transform.react.throwIfNamespace
 
-- `throwIfNamespace`
-  Toggles whether or not to throw an error if an XML namespaced tag name is used. For example: `<f:image />`
+Toggles whether or not to throw an error if an XML namespaced tag name is used. For example: `<f:image />`
 
 Though the JSX spec allows this, it is disabled by default since React's JSX does not currently have support for it.
 
-- `development`
-  Toggles plugins that aid in development, such as `jsx-self` and `jsx-source`. Automatically read `webpack` mode when used with `swc-loader`
+#### jsc.transform.react.development
+
+Toggles debug props `__self` and `__source` on elements generated from JSX, which are used by development tooling such as React Developer Tools.
+
+Automatically read `webpack` mode when used with `swc-loader`.
```

Comment:
I wasn't sure what exactly this sentence refers to, so I didn't change it. However, it is unclear to me still.

Reply:
I meant, if webpack is in development mode, jsx debug transforms are enabled.